### PR TITLE
fix: prepare api client for dashboard

### PR DESCRIPTION
**Pull request by cameronrohani**
we won't be having any sidenav's across the app with the new dashboard, so I've had to make quite a lot of changes to remedy this

image

image

Also made a small handful of visual tweaks

### CODE REVIEW

File: packages/api-client/src/components/SideNav/SideNav.vue [hunk 1]
```diff
@@ -12,14 +10,13 @@ const { currentRoute } = useRouter()
 <template>
   <nav
     aria-label="Side Navigation"
-    class="text-c-2 w-15 flex flex-col items-center px-2 py-2 scalar-sidenav relative drag-region"
+    class="text-c-2 w-15 flex items-center scalar-sidenav relative no-drag-region mr-2"
     role="navigation">
-    <WorkspaceProfileIcon class="no-drag-region" />
-    <ul class="mt-3.5 flex flex-col gap-1.5">
+    <!-- <WorkspaceProfileIcon class="no-drag-region" /> -->
```
Comment (antlio) [inline packages/api-client/src/components/SideNav/SideNav.vue:15]: should we maintain this while using the app but not in the dashboard maybe?

File: packages/api-client/src/components/ViewLayout/ViewLayoutSection.vue [hunk 2]
```diff
@@ -9,7 +9,7 @@ const id = nanoid()
     class="flex xl:min-w-0 xl:flex-1 flex-col xl:custom-scroll bg-b-1">
     <div
       :id="id"
-      class="xl:min-h-header py-2.5 flex items-center border-b-1/2 px-4 xl:px-6 text-sm font-medium sticky top-0 bg-b-1 z-20 rounded-t xl:rounded-none">
+      class="py-2.5 flex items-center border-b-1/2 px-4 xl:px-6 text-sm font-medium sticky top-0 bg-b-1 z-20 rounded-t xl:rounded-none xl:min-h-header">
```
Comment (antlio) [inline packages/api-client/src/components/ViewLayout/ViewLayoutSection.vue:12]: should we apply `xl:px-5` to maintain alignment with lower table?

**Comment by hanspagel** [comment]
Supercool! Not a huge fan of the tiny top navigation, didn’t even see it for minutes. 👀

**Comment by cameronrohani** [comment]
> Supercool! Not a huge fan of the tiny top navigation, didn’t even see it for minutes. 👀

image

thoughts @hanspagel  @antlio (rough prototype would make tabs connected like in chrome)

**Comment by hanspagel** [comment]
TBH, I’m not a big fan of this solution:

Screenshot 2024-09-18 at 12 29 11

My humble opinion: It looks a little foreign to the other UI and it doesn’t give me the impression of what it actually is. It looks more like … I don't know a toggle between production/development or something like that. 🤔

File: packages/oas-utils/test/fixtures/basic-circular-spec.json
Comment (hanspagel) [inline packages/oas-utils/test/fixtures/basic-circular-spec.json:1]: Wait … what’s going on here. The Git history seems to be off.

**Comment by hanspagel** [comment]
Closing because of the amount of conflicts 🥹

**Pull request closed without merging**